### DISCUSSION TITLE
Automatic backups

### DISCUSSION THREAD

**Discussion by galantra** [astashov/liftosaur#174] [Ideas]
Run "Export data to JSON" to a user-defined location at regular intervals (user-defined or simply every day).

**Comment by yrinkel** [comment]
The web editor (and possibly the app) already does a regular backup when you edit.

**Reply by astashov** [reply]
Only for programs though... But also it does backups on the server, so likely will close it as wontfix

**Comment by astashov** [comment]
That's kinda the purpose of having an account - it backups data to the server...
